### PR TITLE
chore: (yahcli) Use hash of gRPC cert PEM instead of its X.509 DER encoding

### LINKED ISSUE #16723: yahcli should use UTF-8 encoding of hexed cert hash in DAB updates
### Description

On using yahcli to perform a `NodeUpdate` or `NodeCreate` the gRPC cert hashes are computed as the SHA-384 hash of the X.509 DER encoding.

However, the legacy representation of a gRPC cert hash in `0.0.101/102` uses the hash of the UTF-8 encoding of a PEM file _containing_ this certificate. So we should preserve this behavior.

### Steps to reproduce

As above.

### Additional context

_No response_

### Hedera network

_No response_

### Version

multiple

### Operating system

None

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Closes #16723

### CODE REVIEW

**Review by jsync-swirlds — APPROVED** [review]

File: hedera-node/test-clients/yahcli/run/publish.sh [hunk 1]
```diff
@@ -25,5 +25,5 @@ cd "${SCRIPT_PATH}/.."
 rm -f assets/yahcli.jar >/dev/null 2>&1 || true
 cp -f yahcli.jar assets/
 
-docker buildx create --use --name "multiarch$TAG"
+docker buildx create --use --name "multiarchASDF"
```
Comment (mhess-swl) [inline hedera-node/test-clients/yahcli/run/publish.sh:28]: Intentional, right?

**Review by mhess-swl — APPROVED** [review]
@tinker-michaelj of course you got pulled into another yahcli ticket. Sorry I missed this.

File: hedera-node/test-clients/src/yahcli/java/com/hedera/services/yahcli/commands/nodes/CreateCommand.java [hunk 2]
```diff
@@ -126,6 +133,14 @@ public Integer call() throws Exception {
         return 0;
     }
 
+    static byte[] allBytesAt(@NonNull final Path path) {
+        try {
+            return Files.readAllBytes(path);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
```
Comment (ElijahLynn) [inline hedera-node/test-clients/src/yahcli/java/com/hedera/services/yahcli/commands/nodes/CreateCommand.java:142]: Copilot had a suggestion:

```suggestion
    static byte[] allBytesAt(@NonNull final Path path) {
        try {
            return Files.readAllBytes(path);
        } catch (IOException e) {
            throw new FileReadException("Failed to read all bytes from file at path: " + path, e);
        }
    }
```

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by ElijahLynn — APPROVED** [review]
tested and it works!

**Pull request merged**